language-plutus-core/src/Language/PlutusCore/Pretty.hs

```diff
@@ -59,48 +54,15 @@ module Language.PlutusCore.Pretty
     , PrettyReadable
     , topPrettyConfigReadable
     , botPrettyConfigReadable
+    -- * Utils
+    , prettyBytes
     ) where
 
-import           Language.PlutusCore.Name            as Export
-import           Language.PlutusCore.Pretty.Classic  as Export
-import           Language.PlutusCore.Pretty.Plc      as Export
-import           Language.PlutusCore.Pretty.Readable as Export
-import           Language.PlutusCore.Type
 import           PlutusPrelude
 
-import           Data.Text                           (Text)
-
--- | Pretty-print a value in the default mode using the classic view.
-prettyPlcDef :: PrettyPlc a => a -> Doc ann
-prettyPlcDef = prettyPlcClassicDef
```

Comment:
The top-level `Pretty` module now doesn't define anything and instead reexports stuff from other modules.